packages/@aws-cdk/aws-eks/lib/helm-chart.ts

```diff
@@ -0,0 +1,116 @@
+import { CustomResource, CustomResourceProvider } from '@aws-cdk/aws-cloudformation';
+import * as lambda from '@aws-cdk/aws-lambda';
+import { Construct, Duration, Stack } from '@aws-cdk/core';
+import * as path from 'path';
+import { Cluster } from './cluster';
+import { KubectlLayer } from './kubectl-layer';
+
+/**
+ * Helm Chart Properties.
+ */
+export interface HelmChartProps {
+  /**
+   * The EKS cluster to apply this configuration to.
+   *
+   * [disable-awslint:ref-via-interface]
+   */
+  readonly cluster: Cluster;
+
+  /**
+   * The name of the release.
+   */
+  readonly name: string;
+
+  /**
+   * The name of the chart.
+   */
+  readonly chart: string;
+
```

Comment:
What's the difference between `name` and `chart`. Can we deduce name from `construct.node.uniqueId`?

Reply:
The difficulty is that there is a name for the chart and a name for the release. The release name is optional and can be generated.